Polygon_repair_2/include/CGAL/Polygon_repair_2/Polygon_repair_2.h

```diff
@@ -0,0 +1,470 @@
+// Copyright (c) 2023 GeometryFactory.
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Ken Arroyo Ohori
+//                 Guillaume Damiand
+
+#ifndef CGAL_POLYGON_REPAIR_2_H
+#define CGAL_POLYGON_REPAIR_2_H
+
+#include <CGAL/license/Polygon_repair_2.h>
+
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
```

Comment:
do you really need that include?

Reply:
Only if we want to have that kernel as a default template parameter, I think?

Reply:
we should get it from the polygon, shouldn't we?